tests/integration_tests/charts/data/api_tests.py

```diff
@@ -1188,10 +1193,10 @@ def test_data_cache_default_timeout(
 
 
 def test_chart_cache_timeout(
+    load_energy_table_with_slice: List[Slice],
```

Comment:
This is an interesting case that surfaced on this new Flask version. Flask now is stricter, so when popping a new context from the app stack it checks if the context is the one we are expecting. `load_energy_table_with_slice` creates a new `app_context` so it needs to be first